templates/_macros/hero.jinja

```diff
@@ -0,0 +1,111 @@
+# Params
+# title: Hero title text (required)
+# subtitle: Hero subtitle text
+# layout: layout of hero section. Options are '50/50', '50/50-full-width-image', '75/25', '25/75', 'fallback'
+# is_split_on_medium: whether the layout is split on tablet. Applies "-on-medium" to row class in 50/50 and 25/75 layouts. If false, the layout is stacked on tablet.
+# Slots
+# description: paragraph-style content below the title and subtitle
+# cta: call-to-action block below the description
+# image: slot for image content
+
+{% macro hero(
+  title='',
```

Comment:
Can we enforce passing title, by not setting the default value (not sure if that works this way in macros)?

Reply:
I'll try it out for the 25-75 PR!